web/components.d.ts

```diff
@@ -77,6 +77,7 @@ declare module '@vue/runtime-core' {
     IPhXCircle: typeof import('~icons/ph/x-circle')['default']
     ISimpleIconsGitea: typeof import('~icons/simple-icons/gitea')['default']
     ITeenyiconsGitSolid: typeof import('~icons/teenyicons/git-solid')['default']
+    ITeenyiconsRefreshOutline: typeof import('~icons/teenyicons/refresh-outline')['default']
```

Comment:
@anbraten What's this change about?

Reply:
@lukashass Its an automatically created file adding typings for all components. It has some issues detecting the icon components, so each time you open a page or so with an icon not being indexed before it adds that component. We just approve it in reviews. 🙈